<?xml version="1.0"?>
<robot name="parameters" xmlns:xacro="http://www.ros.org/wiki/xacro">

        <!-- See README.md for further explanations -->
        <config>
                <param name="2.3.position_controller">-1.0e+6 1.0e+6 40.0e+3 1.00 1.0e-3 -2000.0 2000.0 0.2 0.17 1600</param>
                <param name="2.3.current_controller">-688.0 688.0 0.405 10.0e-6 -0.125 0.125 0 1984 -248.0 248.0</param>
                <param name="2.3.home_settings">-1 2.0e+3 42.0e+3 -8.0e+3 0.8 0.85</param>
                <param name="2.1.position_controller">-1.0e+6 1.0e+6 40.0e+3 1.00 1.0e-3 -2000.0 2000.0 0.2 0.17 1600</param>
                <param name="2.1.current_controller">-688.0 688.0 0.405 10.0e-6 -0.125 0.125 0 1984 -248.0 248.0</param>
                <param name="2.1.home_settings">-1 2.0e+3 42.0e+3 -8.0e+3 0.8 0.75</param>
                <param name="1.2.position_controller">-1.0e+6 1.0e+6 40.0e+3 1.00 1.0e-3 -500.0 500.0 0.8 0.0 1000.0</param>
                <param name="1.2.current_controller">-350.0 350.0 0.405 4.0e-6 -25.0 25.0 1 10.0 -255.0 255.0</param>
                <param name="1.2.home_settings">-1 2.0e+3 42.0e+3 -8.0e+3 0.8 0.75</param>
                <param name="0.0.position_controller">-1.0e+6 1.0e+6 40.0e+3 1.00 1.0e-3 -500.0 500.0 0.8 0.0 1000.0</param>
                <param name="0.0.current_controller">-350.0 350.0 0.405 4.0e-6 -25.0 25.0 1 10.0 -255.0 255.0</param>
                <param name="0.0.home_settings">-1 2.0e+3 42.0e+3 -8.0e+3 0.8 0.75</param>
        </config>
</robot>
